libs/github/src/lib/github.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { HttpClientModule } from '@angular/common/http';
+import { GithubService } from './github.service';
```

Comment:
I don't think we should be creating a new lib just for this service, I'd move this to the utils lib

Reply:
Yeah, agree. But I was thinking of reuse create/close issue method. May be I'll add them here and rename this GithubService lib name to something more proper